src/main/java/racingcar/domain/RacingCars.java

```diff
@@ -0,0 +1,50 @@
+package racingcar.domain;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class RacingCars {
+    private List<Car> cars;
+
+    public RacingCars(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public static RacingCars of(String carNameText) {
+        return new RacingCars(Arrays.stream(carNameText.split(","))
+                .map(Car::new)
+                .collect(Collectors.toList()));
+    }
+
+    public int size() {
+        return cars.size();
+    }
+
+    public void race() {
+        this.cars.forEach(Car::run);
+    }
+
+    public Winners pickWinners() {
+        Collections.sort(cars);
+
+        Car firstWinner = cars.get(0);
+
+        List<Car> winners = cars.stream()
+                .filter(car -> car.compareTo(firstWinner) == 0)
```

Comment:
메서드 이름을 조금 더 명확하고 비즈니스 로직에 가깝게 지어보는 건 어떨까요?
```suggestion
                .filter(car -> car.hasSamePosition(firstWinner))
```